test/model/enc_frame.py

```diff
@@ -164,21 +165,90 @@ def zigzag(block):
         r[zig_zag_order[i]] = block[i]
     return r
 
-#def rle_code(block):
-#    entropy = []
-#    count = 0
-#    for i in block:
-#        if()
-
-#    return entropy
-
 def rle_code(block):
     return list(chain(*[(symbol,len(list(g))) for symbol, g in groupby(block)]))
 
+def build_huffman_rom_tables(csvfile):
+    """build huffman tables"""
+    code = []
+    size = []
+    with open(csvfile, 'r') as csvfp:
+        csvreader = csv.reader(csvfp, delimiter=',')
+        for row in csvreader:
+            code.append(row[0])
+            size.append(row[1])
+    code = tuple(code)
+    size = tuple(size)
+    return code, size

```

Comment:
This function seems to appear twice.

Reply:
@mithro , I am not able to find the function twice.